src/plugins/intel_gna/src/serial/gna_model_serial.cpp

```diff
@@ -44,8 +44,8 @@ inline void writeBits(const T& obj, std::ostream& os) {
 
 inline void writeString(const std::string& str, std::ostream& os) {
     const char* c_str = str.c_str();
-    const size_t str_len = strlen(c_str) + 1;
-    writeBits(static_cast<uint32_t>(str_len), os);
+    const uint32_t str_len = static_cast<uint32_t>(strlen(c_str)) + 1;
+    writeBits(str_len, os);
```

Comment:
don't change a type of variable passes to writeBits or writeNBytes

Reply:
I didn't change the type - before, str_len was size_t but it was casted to uint32_t as an argument to writeBits. Now it's declared as uint32_t to avoid unnecessary cast to uint32_t in the next line.

writeBits was and is called with uint32_t; writeNBytes accepts only uint32_t, so it was called with size_t with implicit conversion to uint32_t; now it's called explicitly with uint32_t.

Nothing changed here.